src/init/VersionInitializer.ts

```diff
@@ -0,0 +1,29 @@
+import { readJson } from 'fs-extra';
+import type { KeyValueStorage } from '../storage/keyvalue/KeyValueStorage';
+import { modulePathPlaceholder, resolveAssetPath } from '../util/PathUtil';
+import { Initializer } from './Initializer';
+
+const STORAGE_KEY = 'current-server-version';
```

Comment:
Mostly because the other one had a direct link to a variable and I this reduced the amount of inspiration I needed. 😄 Changing the other one to something like `current-base-url` then.